zetaclient/query.go

```diff
@@ -214,6 +232,16 @@ func (b *ZetaCoreBridge) GetAllOutTxTrackerByChain(chain common.Chain) ([]types.
 	if err != nil {
 		return nil, err
 	}
+	if order == Ascending {
+		sort.SliceStable(resp.OutTxTracker, func(i, j int) bool {
+			return resp.OutTxTracker[i].Nonce < resp.OutTxTracker[j].Nonce
+		})
+	}
+	if order == Descending {
+		sort.SliceStable(resp.OutTxTracker, func(i, j int) bool {
+			return resp.OutTxTracker[i].Nonce > resp.OutTxTracker[j].Nonce
+		})
+	}
```

Comment:
Maybe we should return an error if none of those

Reply:
Though on this?
Do we skip sorting we none of Descending or Ascending is provided?
In the code we have
```
const (
	Ascending  Order = "ASC"
	Descending Order = "DESC"
)
```
I would suggest to have 
```
NoOrder Order = ""
```
to make it explicit that the zero value is no ordering